tamr_client/categorization/project.py

```diff
@@ -0,0 +1,52 @@
+from tamr_client._types import (
+    CategorizationProject,
+    Dataset,
+    Instance,
+    JsonDict,
+    Session,
+    URL,
+)
+from tamr_client.dataset import _dataset, unified
+
+
+def _from_json(url: URL, data: JsonDict) -> CategorizationProject:
+    """Make Categorization project from JSON data (deserialize)
+
+    Args:
+        url: Project URL
+        data: Project JSON data from Tamr server
+    """
+    return CategorizationProject(
+        url, name=data["name"], description=data.get("description")
+    )
+
+
+def manual_labels(
+    session: Session, instance: Instance, project: CategorizationProject
+) -> Dataset:
+    """
+    Get manual labels from a Categorization project
+    Args:
+        instance: Tamr instance containing this dataset
+        project: Tamr project of this Unified Dataset
+
+    Returns:
+        Dataset containing manual labels
+
+    """
+    unified_dataset = unified.from_project(
+        session=session, instance=instance, project=project
+    )
+    labels_dataset_name = unified_dataset.name + "_manual_categorizations"
+    dataset_url = URL(
```

Comment:
I like where this code is going.  This section highlights that TC does not have functionality to get a dataset by name and this is a good (and versioned) way of doing so.  I think it would be worth pulling this out into a function itself, like `by_name` in `tamr_client/dataset/_dataset.py`.  Could be part of this PR, or just noted as a followup.